user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Feeds/Specifying_data_input_in_a_dashboard_URL.md

```diff
@@ -106,6 +115,25 @@ To pass this JSON object as part of a URL, it needs to be URL-encoded.
 > - ] is encoded as %5D.
 > - / is encoded as %2F.
 
+#### Example 2: low-code app
+
+The following example URL selects one default element on the initial page. The component ID is "1", and the element ID is "1/6"
+
+```url
+https://<dma>/<app-id>?data={"version":1,"components":[{"cid":1,"select":{"elements":["1/6"]}]}}
+```
+
+This is the encoded equivalent: `https://<dma>/<app-id>?data=%7B%22v%22:1,%22components%22:%5B%7B%22cid%22:1,%22select%22:%7B%22elements%22:%5B%221%2F6%22%5D%7D%5D%7D%7D`
+
+> [!NOTE]
+>
+> - { is encoded as %7B
+> - " is encoded as %22.
+> - [ is encoded as %5B.
+> - ] is encoded as %5D.
+> - / is encoded as %2F.
+> - } is encoded as %7D
```

Comment:
depending on what data is selected, you can also have a space " " which gets encoded as %20.
Eg. "select":{"object manager instances":["guid"]}
becomes: %22select%22:%7B%22object%20manager%20instances%22:%5B%22guid%22%5D%7D

Reply:
Instead of adding a list of the characters and their URL-encoded equivalents used in the examples, I've added a new section to add this information (including space = %20). See [commit 705d0de](https://github.com/SkylineCommunications/dataminer-docs/pull/3100/commits/705d0de66fea55733154d2596bd4d003dd89ca5f)

Reply:
Even better!